.storybook/coveSbThemes.js

```diff
@@ -19,7 +19,7 @@ export const managerTheme = create({
   appBorderRadius: 0,
 
   // Typography
-  fontBase: '"Open Sans", sans-serif',
+  fontBase: '"Nunito", sans-serif',
```

Comment:
nit: in this file I'm wondering if we could remove the typography settings in favor of importing CSS. If its overly complicated this is fine, especially since its Storybook.
nit: convert previous js files to ts files when possible.

Reply:
Partially resolved: I changed this file to typescript but so far I wasn't able to figure out a theming approach using imported css - going to put that enhancement off for now in favor of getting this pr in so we can begin other UX tickets